### DISCUSSION TITLE
Default model type used

### DISCUSSION THREAD

**Discussion by iknoorjobs** [PaddlePaddle/PaddleOCR#13838] [Q&A]
Hi, 
When I use the Python library to do OCR, what are the default models (or algorithms) used for detection and recognition? For eg.
```
from paddleocr import PaddleOCR, draw_ocr
ocr = PaddleOCR(
    use_gpu=False, 
    use_angle_cls=True, 
    lang='en', 
)
result = ocr.ocr(image_path, cls=True)
```
Also, if I set `lang` to other languages (eg. Arabic), which model is used for text detection and recognition in those cases? Where to find this information?

Thanks

**Comment by GreatV** [comment]
please refer to the original source code

https://github.com/PaddlePaddle/PaddleOCR/blob/5b54ac460623d7dd23717e39b43b182a359bd606/paddleocr.py#L88-L328
